start/sw/web-app/laravel5.md

```diff
@@ -9,46 +9,53 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
```

Comment:
This is just a matter of preference, right? Could you include both options then?

Reply:
Actually no, Laravel Valet uses Nginx

Reply:
Ok, then please mention this in the quickstart.